### PR TITLE
✨ add pool title bar

### PR DESCRIPTION
**Pull request by okusindja**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206042570035492

### CODE REVIEW

**Review by MarioBatalha — APPROVED** [review]

File: views/pools/index.tsx [hunk 1]
```diff
@@ -2,13 +2,25 @@ import { Typography } from '@interest-protocol/ui-kit';
...
+import { DefaultTokenSVG } from '@/svg';
+
+import PoolTitleBar from './pool-title-bar';
 
 const Pools: FC = () => {
   return (
     <Layout>
       <Typography my="2xl" size="large" variant="display" textAlign="center">
         Pools
       </Typography>
```
Comment (KipandaJr) [inline views/pools/index.tsx:14]: Remove this title

File: views/pools/pool-title-bar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          {poolName}
+        </Typography>
+        <Box ml="auto" gap="1rem" display="flex" alignItems="center">
+          <Box width="2.5rem" height="2.5rem">
+            {poolExitCoin}
+          </Box>
+          <Box width="2.5rem" height="2.5rem">
+            {poolEnterCoin}
+          </Box>
+        </Box>
```
Comment (KipandaJr) [inline views/pools/pool-title-bar/index.tsx:39]: This box should be ride on mobile

File: views/pools/pool-title-bar/pool-title-bar.types.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { ReactNode } from 'react';
+
+export interface PoolTitleBarProps {
+  poolName: string;
+  poolEnterCoin: ReactNode;
+  poolExitCoin: ReactNode;
```
Comment (KipandaJr) [inline views/pools/pool-title-bar/pool-title-bar.types.ts:6]: why this name like `poolEnterCoin` or `poolExitCoin`?

File: views/pools/pool-title-bar/pool-title-bar.types.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { ReactNode } from 'react';
+
+export interface PoolTitleBarProps {
```
Comment (KipandaJr) [inline views/pools/pool-title-bar/pool-title-bar.types.ts:3]: miss the function to call on Back button

**Review by KipandaJr — CHANGES_REQUESTED** [review]
Check on mobile ui, are weird.

**Pull request merged**